2024-09-01 18:23:10 - Error al insertar en Pipedrive: 'NoneType' object has no attribute 'get'
2024-09-01 18:30:45 - Error al insertar en Pipedrive: 'NoneType' object has no attribute 'get'
2024-09-01 18:31:06 - Error al insertar en Pipedrive: 'NoneType' object has no attribute 'get'
2024-09-01 18:31:52 - Error al insertar en Pipedrive: 'NoneType' object has no attribute 'get'
2024-09-03 13:45:39 - Error al insertar o actualizar el cliente 'C3452945': manejar_no_existencia_campo() takes 3 positional arguments but 4 were given
2024-09-03 13:49:44 - Error al insertar o actualizar el cliente 'C3452945': manejar_no_existencia_campo() takes 3 positional arguments but 4 were given
2024-09-04 08:08:13 - Error al insertar o actualizar el cliente 'C3452945': manejar_no_existencia_campo() takes 3 positional arguments but 4 were given
2024-09-04 08:15:46 - Error al insertar o actualizar el cliente 'C3452945': manejar_no_existencia_campo() takes 3 positional arguments but 4 were given
2024-09-04 08:15:48 - Error al insertar o actualizar el cliente 'C3452945': manejar_no_existencia_campo() takes 3 positional arguments but 4 were given
2024-09-04 08:15:49 - Error al insertar o actualizar el cliente 'C3452945': manejar_no_existencia_campo() takes 3 positional arguments but 4 were given
2024-09-04 08:15:51 - Error al insertar o actualizar el cliente 'C3452945': manejar_no_existencia_campo() takes 3 positional arguments but 4 were given
2024-09-04 08:21:13 - Error al insertar o actualizar el cliente 'C3452945': manejar_no_existencia_campo() takes 3 positional arguments but 4 were given
2024-09-04 08:21:15 - Error al insertar o actualizar el cliente 'C3452945': manejar_no_existencia_campo() takes 3 positional arguments but 4 were given
2024-09-04 08:23:57 - Error al insertar o actualizar el cliente 'C3452945': manejar_no_existencia_campo() takes 3 positional arguments but 4 were given
2024-09-04 08:23:59 - Error al insertar o actualizar el cliente 'C3452945': manejar_no_existencia_campo() takes 3 positional arguments but 4 were given
2024-09-04 20:27:43 - Error al conectar a la base de datos: cannot access local variable 'count' where it is not associated with a value
2024-09-04 20:29:36 - Error al conectar a la base de datos: cannot access local variable 'count' where it is not associated with a value
2024-09-04 21:39:17 - La cotización con DocNum: '17860' y DocEntry: '23231' tiene el siguiente error: 'validado'
2024-09-04 21:39:26 - La cotización con DocNum: '17864' y DocEntry: '23236' tiene el siguiente error: 'validado'
2024-09-04 21:41:37 - La cotización con DocNum: '17860' y DocEntry: '23231' tiene el siguiente error: 'validado'
2024-09-04 21:41:46 - La cotización con DocNum: '17864' y DocEntry: '23236' tiene el siguiente error: 'validado'
2024-09-04 21:42:35 - La cotización con DocNum: '17860' y DocEntry: '23231' tiene el siguiente error: 'validado'
2024-09-04 22:22:13 - HTTPError in post_request: 400 Client Error: Bad Request for url: https://api.pipedrive.com/v1/deals?api_token=bead2d5c759ded1c92f16cd8cee80ff8e1b8e82d
2024-09-04 22:22:13 - No se pudo insertar el deal en Pipedrive.
2024-09-04 22:28:42 - HTTPError in post_request: 400 Client Error: Bad Request for url: https://api.pipedrive.com/v1/deals?api_token=bead2d5c759ded1c92f16cd8cee80ff8e1b8e82d
2024-09-04 22:28:42 - No se pudo insertar el deal en Pipedrive.
2024-09-04 22:46:35 - HTTPError in post_request: 400 Client Error: Bad Request for url: https://api.pipedrive.com/v1/deals?api_token=bead2d5c759ded1c92f16cd8cee80ff8e1b8e82d
2024-09-04 22:46:35 - No se pudo insertar el deal en Pipedrive.
2024-09-04 22:54:03 - HTTPError in post_request: 400 Client Error: Bad Request for url: https://api.pipedrive.com/v1/deals?api_token=bead2d5c759ded1c92f16cd8cee80ff8e1b8e82d
2024-09-04 22:54:03 - No se pudo insertar el deal en Pipedrive.
2024-09-04 23:22:34 - HTTPError in post_request: 400 Client Error: Bad Request for url: https://api.pipedrive.com/v1/deals?api_token=bead2d5c759ded1c92f16cd8cee80ff8e1b8e82d
2024-09-04 23:22:34 - No se pudo insertar el deal en Pipedrive.
2024-09-04 23:24:19 - Error en el proceso de inserción de deal y actualización de tablas: 'list' object has no attribute 'update'
2024-09-04 23:25:42 - HTTPError in post_request: 400 Client Error: Bad Request for url: https://api.pipedrive.com/v1/deals?api_token=bead2d5c759ded1c92f16cd8cee80ff8e1b8e82d
2024-09-04 23:25:42 - No se pudo insertar el deal en Pipedrive.
